<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDlg</class>
 <widget class="QDialog" name="InputDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_point">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改值:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
